server/_server/models.py

```diff
@@ -1,7 +1,55 @@
+from anytree import NodeMixin, RenderTree, PreOrderIter
+
+
+class Employee(NodeMixin):
+    """Tree-Node functionallity
+    Base model for building the OrgChart
+
+    examples;
+        <jhon@tuenti.com: 1>
+        <jane@tuenti.com (business development): 3>
+    """
+
+    def __init__(self, mail: str, area: str = None, supervisor=None, minions=None):
+        assert '@' in mail
+        super().__init__()
+        self.mail = mail
+        self.area = area
+        self.parent = supervisor
+        if minions:
+            self.children = minions
+
+    @property
+    def name(self) -> str:
+        return self.mail.split('@')[0]
+
+    @property
+    def minions(self) -> int:
+        return len(self.children)
```

Comment:
mmm this method for me it seems like it returns a list of minions and not how many minions it has, What about to just have one methos to retrieve the list of minions? the length could be computed anywherte else

Reply:
actually you can do 

```python
org_chart.children
# => [node, node, node]
```

while 

```python
org_chart.minions
# => 3
```

Reply:
now I see this, I agree with you. Renaming it :)